docs/images/sc-release-process.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It would be nice to start Docker and Helm with uppercase. Also: API Server.

Reply:
fixed!